fast64_internal/f3d/f3d_bleed.py

```diff
@@ -45,9 +61,61 @@
     GfxList,
     FTriGroup,
     GbiMacro,
+    get_F3D_GBI,
 )
 
 
+def get_geo_cmds(
+    clear_modes: set[str], set_modes: set[str], is_ex2: bool, matWriteMethod: GfxMatWriteMethod
+) -> tuple[
+    list[SPLoadGeometryMode | SPGeometryMode | SPSetGeometryMode | SPClearGeometryMode]
+    | list[SPGeometryMode | SPSetGeometryMode | SPClearGeometryMode]
```

Comment:
I think you meant `tuple[A, B]` ?
```suggestion
    , list[SPGeometryMode | SPSetGeometryMode | SPClearGeometryMode]
```
\+ formatting